### DISCUSSION TITLE
Enable batch mode in splunk plugin

### DISCUSSION THREAD

**Discussion by blackumbrella** [fluent/fluent-bit#6468] [Q&A]
In recent splunk integration, we noticed the fluentbit splunk plugin doesn't support batch mode which allows to send mulitple log entries in one HTTP request which can save a lot of HTTP connections but interesting part is fluentd plugin has provided batch supports.

is it a good idea to get same feature into FluentBit? This feature actually should be very important to enterprise or big customers which can generate tons of logs every day.

refer to: https://github.com/splunk/fluent-plugin-splunk-hec

**Comment by agup006** [comment]
Hmm we batch in chunks of 2MB by default @blackumbrella , is there some specific batch behavior you are not seeing or are looking for?
